geocoding_platform_interface/analysis_options.yaml

```diff
@@ -1,4 +1,4 @@
-include: package:effective_dart/analysis_options.yaml
+include: package:pedantic/analysis_options.yaml
```

Comment:
Please rollback this change since we prefer the use of the `effective_dart` style guide.